src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars__star stars--0"></div>
```

Comment:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
```
you should add modifier for the parent block, fix it everywhere